### PR TITLE
Basic events list

### PR DESCRIPTION
**Pull request by ShayDavidson**
pp

### CODE REVIEW

File: app/src/main/java/com/paypal/psix/adapters/EventsAdapter.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import butterknife.InjectView;
+
+/**
+ * Created by shay on 3/3/15.
+ */
+public class EventsAdapter extends ArrayAdapter<Event> {
+
+    static class ViewHolder {
+        @InjectView(R.id.event_image) ImageView image;
+        @InjectView(R.id.event_name) TextView label;
```
Comment (ShayDavidson) [inline app/src/main/java/com/paypal/psix/adapters/EventsAdapter.java:26]: butterknife awesomeness in action (http://jakewharton.github.io/butterknife/).
no need to declare these in the code flow

File: app/src/main/java/com/paypal/psix/fragments/EventsFragment.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    @InjectView(R.id.events_list_view) ListView listView;
+
+    public EventsFragment() {
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.fragment_events, container, false);
+        ButterKnife.inject(this, rootView);
```
Comment (ShayDavidson) [inline app/src/main/java/com/paypal/psix/fragments/EventsFragment.java:39]: needed (also in `onDestroyView`) on fragments and butterknife.

File: app/src/main/java/com/paypal/psix/activities/EventsActivity.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+//        } else if (id == R.id.action_debug_setup_event) {
+//            startActivity(new Intent(this, SetupEventActivity.class));
+//            return true;
+//        } else if (id == R.id.action_debug_events_list) {
+//            startActivity(new Intent(this, EventsActivity.class));
+//        }
+//
+//        return super.onOptionsItemSelected(item);
+//    }
+}
```
Comment (gardenofwine) [inline app/src/main/java/com/paypal/psix/activities/EventsActivity.java:44]: remove comments

Comment (ShayDavidson) [inline app/src/main/java/com/paypal/psix/activities/EventsActivity.java:44]: pushed by mistake, working on it now :)

File: app/src/main/java/com/paypal/psix/adapters/EventsAdapter.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    public EventsAdapter(Context context, ArrayList<Event> events) {
+        super(context, 0, events);
+    }
+
+
+    @Override
+    public View getView(int position, View view, ViewGroup parent) {
+        Event event = getItem(position);
+
+        ViewHolder holder;
```
Comment (gardenofwine) [inline app/src/main/java/com/paypal/psix/adapters/EventsAdapter.java:42]: Why not put this code inside the ViewHolder?

Comment (ShayDavidson) [inline app/src/main/java/com/paypal/psix/adapters/EventsAdapter.java:42]: how? all the methods used here are on the fragment only (`getContext`, the view param).

**Comment by gardenofwine** [comment]
:ok:

**Pull request merged**